arm/Microsoft.Sql/serversResources/databases/deploy.bicep

```diff
@@ -0,0 +1,95 @@
+@description('Optional. The collation of the database.')
```

Comment:
So are we removing features (e.g. PE)?

Reply:
I did remove it here because the implementation was very lackluster and in no way consistent with our other implementations & how the portal does it. It'd much rather see it re-implemented than carry it over